### PR TITLE
add `twicpics`

### PR DESCRIPTION
**Pull request by florentb**
Current doc: https://github.com/TwicPics/components/blob/0.6.2/documentation/nuxt2.md

### CODE REVIEW

**Comment by mbgspcii** [comment]
> Current doc: https://github.com/TwicPics/components/blob/0.6.2/documentation/nuxt2.md

Latest versions of the documentation are available here :
- https://github.com/TwicPics/components/blob/main/documentation/nuxt2.md
- https://github.com/TwicPics/components/blob/main/documentation/nuxt3.md

File: modules/twicpics.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+name: twicpics
+description: >-
+  Images and videos components - Context aware resizing and cropping,
+  lazy-loading, LQIP, Next-Gen format, compression, CDN.
+repo: TwicPics/components
+npm: '@twicpics/components'
+icon: twicpics.svg
+github: https://github.com/TwicPics/components/tree/main/documentation
+website: https://github.com/TwicPics/components/tree/main/documentation
```
Comment (pi0) [inline modules/twicpics.yml:9]:
```suggestion
website: https://www.twicpics.com/docs/components/nuxt-3
```

File: modules/twicpics.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+name: twicpics
+description: >-
+  Images and videos components - Context aware resizing and cropping,
+  lazy-loading, LQIP, Next-Gen format, compression, CDN.
+repo: TwicPics/components
+npm: '@twicpics/components'
+icon: twicpics.svg
+github: https://github.com/TwicPics/components/tree/main/documentation
```
Comment (pi0) [inline modules/twicpics.yml:8]:
```suggestion
github: https://github.com/TwicPics/components
```

**Review by pi0 — APPROVED** [review]

**Comment by pi0** [comment]
Thanks for PR. Looking at modules both nuxt2 and nuxt3 seems super clean. I've used [website](https://www.twicpics.com/docs/components/nuxt-3) link docs please let me know if prefer to use github directly for docs. Also FYI, `nuxt/image` has built-in twicpics provided now for both Nuxt 2 and Nuxt 3 (https://v1.image.nuxtjs.org/providers/twicpics)

**Pull request merged**